package.json

```diff
@@ -27,12 +27,13 @@
     "prepare:drive:mobile": "(cd src/drive/targets/mobile && rm -rf platforms && rm -rf plugins && cordova prepare)",
     "run:drive:android": "(cd src/drive/targets/mobile && cordova run android --device)",
     "run:drive:android:emulator": "(cd src/drive/targets/mobile && cordova run android --emulator)",
-    "build:drive:android": "npm run build:drive:mobile && (cd src/drive/targets/mobile && cordova build android --release)",
-    "build:drive:android:debug": "npm run build:drive:mobile && (cd src/drive/targets/mobile && cordova build android --debug)",
-    "sign:drive:android": "(cd src/drive/targets/mobile && apksigner sign --ks keys/android/cozy-drive-release-key.jks --out build/android/cozy-drive.apk platforms/android/app/build/outputs/apk/release/app-release-unsigned.apk)",
+    "build:drive:android": "(cd src/drive/targets/mobile && cordova build android --release)",
```

Comment:
Why not using `build:drive:mobile` anymore ?

Reply:
Good question. The steps to build the mobile app are:

```
- yarn build:drive:mobile
- yarn genicon:drive:mobile
- yarn prepare:drive:mobile
- yarn buildsigned:drive:android
```

We *need* to build the mobile app before doing the `prepare` step.
But the `buildsigned` step *also* did a `build:drive:mobile` — which means we had to build the web files twice, which takes a long time.